package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node ./server/index.js",
-    "dev": "run-p dev:server dev:client",
+    "dev": "DB_URL=mongodb://localhost:27017 run-p dev:server dev:client",
```

Comment:
The `DB_URL` should be necessary if you have a `.env` file as described in the README.